### PR TITLE
Fixed Concordance tool to allow for no variation alleles in the truth data.

### PR DESCRIPTION
**Pull request by jonn-smith**
Fixed Concordance tool to allow for no variation alleles in the truth data.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java [hunk 1]
```diff
@@ -291,8 +290,12 @@ public Object onTraversalSuccess() {
     @Override
     protected boolean areVariantsAtSameLocusConcordant(final VariantContext truth, final VariantContext eval) {
         final boolean sameRefAllele = truth.getReference().equals(eval.getReference());
+
         // we assume that the truth has a single alternate allele
-        final boolean containsAltAllele = eval.getAlternateAlleles().contains(truth.getAlternateAllele(0));
+        final boolean containsAltAllele =
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java]: You need to update the comment on line 294.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java [hunk 2]
```diff
@@ -209,4 +212,35 @@ public void testDreamSensitivity() throws Exception {
         // 78
         Assert.assertEquals(snpRecord.getTruePositives() + snpRecord.getFalseNegatives(), 78);
     }
+
+    @Test
+    public void testDoesNotCrashWithNO_VARIATIONAlleles() {
+        final String testDir = toolsTestDir + "concordance/";
```
Comment (davidbenjamin) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java]: This appears elsewhere in this test class, so perhaps extract a static constant for it.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java [hunk 3]
```diff
@@ -209,4 +212,35 @@ public void testDreamSensitivity() throws Exception {
...
+        final String[] args = {
+                "--" + AbstractConcordanceWalker.EVAL_VARIANTS_LONG_NAME, evalVcf.toString(),
+                "--" + AbstractConcordanceWalker.TRUTH_VARIANTS_LONG_NAME, truthVcf.toString(),
+                "--" + Concordance.SUMMARY_LONG_NAME, summary.toString(),
+        };
+
+        runCommandLine(args);
+
+        try {
+            final ConcordanceSummaryRecord.Reader reader             = new ConcordanceSummaryRecord.Reader(summary);
```
Comment (davidbenjamin) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java]: There's some strange spacing here.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java (hunk 1 above)
Comment (jonn-smith) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java]: Fixed!

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java (hunk 2 above)
Comment (jonn-smith) [reply src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java]: Sounds good!  Fixed!

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java (hunk 3 above)
Comment (jonn-smith) [reply src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/ConcordanceIntegrationTest.java]: Ah.  I didn't reformat the code before I checked it in.  I like to align my blocks of variable definitions. 

Fixed!

**Pull request merged**